modules/services/window-managers/river.nix

```diff
@@ -1,16 +1,14 @@
 { config, lib, pkgs, ... }:
-
 let
   inherit (lib) types;
   cfg = config.wayland.windowManager.river;
 
   # Systemd integration
   variables = builtins.concatStringsSep " " cfg.systemd.variables;
-  extraCommands = builtins.concatStringsSep " "
-    (map (f: "&& ${f}") cfg.systemd.extraCommands);
-  systemdActivation = ''
-    ${pkgs.dbus}/bin/dbus-update-activation-environment --systemd ${variables} ${extraCommands}
-  '';
+  systemdActivation = builtins.concatStringsSep " && " ([
+    "${pkgs.dbus}/bin/dbus-update-activation-environment --systemd ${variables}"
+  ] ++ (if cfg.systemd.runInService then [ "systemd-notify --ready" ] else [ ])
```

Comment:
```suggestion
  ] ++ (lib.optional cfg.systemd.runInService "systemd-notify --ready")
```

Reply:
I can't find the doc for this, is it a shortcut for lib.lists.optional?

Reply:
Yes :)